packages/vite-plugin-angular/src/lib/component-resolvers.ts

```diff
@@ -71,6 +71,18 @@ export function hasTemplateUrl(code: string) {
   return code.includes('templateUrl:');
 }
 
+export function getTemplateUrls(code: string) {
+  const project = new Project({ useInMemoryFileSystem: true });
+  const sourceFile = project.createSourceFile('cmp.ts', code);
+  return sourceFile
+    .getDescendantsOfKind(SyntaxKind.PropertyAssignment)
+    .filter((property) => property.getName() === 'templateUrl')
+    .map((property) =>
+      property.getInitializer()?.getText().replace(/['"]/g, '')
```

Comment:
Does this handle this use case for the `templateUrl` if they use `templateUrl: '[param].component.html'` ? 

https://github.com/analogjs/analog/issues/876

Reply:
Yeah, this gets the property value as-is from the object. The replace is to remove the string delimiters which are part of it. 

I can add a unit test for it

Reply:
If you're referring to the scenario in that issue linked, that's regarding `styleUrl`s. I only modified the templateUrl resolver here